less/home-page.less

```diff
@@ -1,61 +1,142 @@
+.content-header {
+  width: 100%;
+  border-bottom: 2px dashed silver;
+
+  h1 {
+    margin-top: 2rem;
+    margin-bottom: -2rem;
+    margin-left: 55rem;
+    color: @font-color;
+
+    @media @mobile {
+      margin-left: 18rem;
+      margin-bottom: 2rem;
+      font-size: 3.5rem;
+    }
+  }
+
+}
+
 .home {
 
+  img {
+    width: 100%;
+    border-radius: 0p;
+
+    @media @mobile {
+      width: 92%;
+      margin-left: 2em;
+      padding-bottom: 2rem;
+    }
+    
+  
+  
+  .top-content {
+    padding-top: 2rem;
+  
+    
+  
+
+    .top-text {
+      padding-bottom: 2rem;
+      border-bottom: 2px dashed silver;
+
+      @media @mobile {
+        width: 92%;
+        margin-left: 2.2rem;
+        padding-bottom: 1rem;
+      }
+
+
+    }
+      h2 {
+        color: @font-color;
+
+        p {
+          color: @font-color;
+        
+        }
+      }
+    }
+  }
+    
+  
+
   .content-section {
     margin: 30px 0;
     display: flex;
     flex-wrap: wrap;
     justify-content: space-between;
+  
+
 
-   
-    
     .text-content {
       width: 48%;
       padding-right: 1%;
-    }
+    
 
-    .img-content {
-      width: 48%;
-      padding-left: 1%;
+    @media @mobile {
+      width: 98%;
+      margin-left: 2rem;
+      padding-bottom: 1rem;
+    
 
-      img {
-        border-radius: 10px;
+      .img-content {
+        width: 48%;
+        padding-left: 1%;
+      
+        img {
+          border-radius: 10px;
+        }
+      }
       }
     }
   }
+    .inverse-content {
+      padding-bottom: 30px;
+      border-bottom: 2px dashed @navigation-border;
+    
+      .text-content {
+        padding-left: 1%;
+        padding-right: 0;
+      
 
-  .inverse-content {
-    padding-bottom: 30px;
-    border-bottom: 2px dashed @navigation-border;
-
-    .text-content {
-      padding-left: 1%;
-      padding-right: 0;
-    }
-
-    .img-content {
-      padding-right: 1%;
-      padding-left: 0;
+      .img-content {
+        padding-right: 1%;
+        padding-left: 0;
+      }
     }
   }
+    
+
+    .content-destination {
+      width: 75%;
+      margin: 0 auto 30px;
 
-  .content-destination {
-    width: 75%;
-    margin: 0 auto 30px;
+    
 
-    img {
-      border-radius: 10px;
+      img {
+        border-radius: 10px;
+      }
     }
-  }
 
-  .content-pick {
-    padding-top: 30px;
-    border-top: 2px dashed @navigation-border;
-    display: flex;
-    justify-content: space-between;
+    .content-pick {
+      padding-top: 30px;
+      border-top: 2px dashed @navigation-border;
+      display: flex;
+      justify-content: space-between;
+
 
-    .destination {
-      width: 30%;
-      margin-bottom: 30px;
+      .destination {
+        width: 30%;
+        margin-bottom: 30px;
+
+
+
+        button {
+          .custom-btn;
```

Comment:
Don't forget to add the parenthesis `()` on the end here since technically, leaving them out is deprecated.